src/content/reference/cli.mdx

```diff
@@ -357,6 +357,53 @@ $ okteto kubeconfig
  i  Updated kubernetes context 'cloud_okteto_com/cindy' in '/Users/cindy/.kube/config'
 ```
 
+### kubetoken
+
+> This command is only available in okteto clusters
+
+Print Kubernetes cluster credentials in [ExecCredential](https://kubernetes.io/docs/reference/config-api/client-authentication.v1/) format. More information on client side authentication can be found [here](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins).
```

Comment:
```suggestion
Prints Kubernetes cluster credentials in [ExecCredential](https://kubernetes.io/docs/reference/config-api/client-authentication.v1/) format. More information on client side authentication can be found [here](https://kubernetes.io/docs/reference/access-authn-authz/authentication/#client-go-credential-plugins).
```

Reply:
I tried following the same tense as the other commands. They are all in infinitive